Add roadmap (#71)
canonical/ulwazi: docs/content/roadmap.md

# Ulwazi roadmap

Ulwazi is under active development.
This roadmap shows the current direction of travel and the major milestones

> we want to reach over the next ~12 months.

Reviewer: ```suggestion
we want to reach over the next by end of year 2026.
```

Reviewer: Adds a time frame of reference; "by end of 26.10 cycle" might be better?

# Ulwazi roadmap

Ulwazi is under active development.
This roadmap shows the current direction of travel and the major milestones
we want to reach over the next ~12 months.

This is a living document for the internal team and all contributors.
Priorities may shift as we learn from adoption and feedback.

> ## Now (26.04)

Reviewer 2: (suggestion, non blocking)

```suggestion
## Ongoing (26.04)
```

Author: I like "Now" better for its better coherence with "Next" and "Later". We can do "Current" as alternative.

These are the goals we are working on in this cycle:

* [x] The theme is published as a Python package
* [x] Contributors’ (developer) documentation is available
* [ ] The theme is validated with code tests
* [ ] The theme is stable enough for wide internal adoption
* [ ] The theme is ready to be used with Starter Pack

> * [ ] Early adopters’ (user) documentation is available

Reviewer 2: (suggestion, non blocking)

```suggestion
* [ ] User documentation for early adopters is available
```

Author: Updated wording for both user and dev docs in a similar way - https://github.com/canonical/ulwazi/pull/71/commits/85f38b5c4a2e08b5c7f7ba01ec8e8973a2f24d8a

* [ ] Plan for switching to Ulwazi as the default theme for the Starter Pack
* [ ] The theme repository is public and ready for external contributions
* [ ] Initial public-theme experience (brand-neutral usage) is supported

## Next (26.10)

These are the goals we plan to work on in the next cycle:

* [ ] The theme is widely used for Canonical's own documentation
* [ ] There is an established feedback process and technical support/maintenance
* [ ] The theme is ready to be used outside Canonical (without Canonical branding)
* [ ] Design system consistency and visual refinement are improved
* [ ] The theme has accessibility checks
* [ ] The theme supports all recommended Sphinx extensions

> * [ ] Full public-theme experience (brand-neutral usage) support

Reviewer 2: Is this different from the third point?

```
* [ ] The theme is ready to be used outside Canonical (without Canonical branding)
```

Author: Good catch! Fixed in  https://github.com/canonical/ulwazi/pull/71/commits/ecfae4bc8edaeb8e8b6601a378a9cb4175ddd066
